doc/semieunit.xml

```diff
@@ -0,0 +1,292 @@
+############################################################################
+##
+#W  semieunit.xml
+#Y  Copyright (C) 2016                                   Christopher Russell
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+<#GAPDoc Label="IsMcAlisterTripleSemigroup">
+  <ManSection>
+    <Prop Name = "IsMcAlisterTripleSemigroup" Arg = "S"/>
+    <Returns><K>true</K> or <K>false</K>.</Returns>
+    <Description>
+      A <E>McAlister triple semigroup</E> is an E-unitary inverse semigroup
+      <Ref Oper= "IsEUnitaryInverseSemigroup" BookName= "Semigroups" />
+      defined by a partial order, a semilattice and a group which acts by
+      order automorphisms on the partial order. All E-unitary inverse
```

Comment:
19. The way it is written now could be interpreted as "there exists a mcalister triple semigroup S such that every e-unitary inverse semigroup is isomorphic to S".